processor/parse/flavored/code-tabs.js

```diff
@@ -25,7 +25,7 @@ function tokenizer(eat, value) {
     kids.push({
       type: 'code',
       className: 'tab-panel',
-      value: code.trim(),
+      value: code.replace(/^\n/, ''),
```

Comment:
I actually think we should just _not_ touch this `code` value at all:
```suggestion
      value: code,
```

Reply:
'cause, like, maybe what if they *want* new lines at the start of a codeblock? it's definitely weird of them, but who am I to judge!

Reply:
Yea, we started by removing the `trim`, cause that seems wrong. But then we found an edge case were the existing regex failed, empty code blocks.

We're only removing the first one. We spent 1.5-ish hours trying to do it in the regex and decided this small change was better.

```
\```\n <-- this one wasn't being removed
\```
```